### PR TITLE
ALB Package updates

### PR DESCRIPTION
**Pull request by jemassey**
* add minimum request count > 1000 on all ALB policies
* add new error policy with static threshold of 5%
* remove request count policies
* change error metric to 5xx metric

### CODE REVIEW

**Review by TheConnMan — CHANGES_REQUESTED** [review]
Could you run this through the Metricly package formatter to fix the Travis builds? (`metricly package format`)

File: policies/high_load_balancer_error_rate.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "conditions": [
+      {
+        "analytic": "actual",
+        "level": 5,
+        "metric": "netuitive.aws.alb.httpcodeelb5xxerrorpercent",
+        "operator": ">"
+      }
+    ],
+    "creatorEmail": "research@metricly.com",
+    "description": "This policy will generate a Critial alert when there are more that 5% http errors to the ALB/Targe Group. You may wish to tune this threshold to better suit your environment.",
```
Comment (TheConnMan) [inline policies/high_load_balancer_error_rate.json]: `ALB/Target Group`

**Review by TheConnMan — CHANGES_REQUESTED** [review]

**Comment by TheConnMan** [comment]
You committed much more than just a spelling fix in your last commit which broke the linting.

**Comment by jemassey** [comment]
Without the lint config file my editor is going to use it's own defaults. Where is the config that metricly-cli is using to validate this against. Is it this one? https://github.com/metricly/metricly-cli/blob/master/tslint.json

**Comment by TheConnMan** [comment]
It's a customer linter, take a look at this PR for background: https://github.com/netuitive-community-packages/netuitive-packages-linux/pull/36

**Review by TheConnMan — APPROVED** [review]

**Pull request merged**